source/shared/net/guides/baker-windows.rst

```diff
@@ -21,24 +21,20 @@ Prerequisites
 Configure a baker node on Windows
 =================================
 
-#. To run a Windows node as baker, you have to select an account as baker account in the Desktop Wallet and generate baker keys for the account. For more information, see :ref:`Add a baker account in the Desktop Wallet<create-baker-desktop>`. You get the account information by exporting a JSON file with the baker credentials from the Desktop Wallet.
+#. Create baker keys and associate them with an account either on mainnet or testnet. We recommend that you do this with the :ref:`Desktop Wallet<create-baker-desktop>`. However, if you want use an account from the mobile wallet as a baker account, you have to use the :ref:`Concordium Client<become-a-baker>`. In the following, the baker keys are referred to as baker-credentials.json.
```

Comment:
Mainnet and testnet are terms used for blockchains in general. So since they are not proper nouns, I'd suggest that we don't capitalize. However, if we're going to use a name such as Concordium Mainnet, we should capitalize.

Reply:
That's a good point! Let's go with that.